### PR TITLE
Remove unused files

### PR DESCRIPTION
**Pull request by wagnerand**
This isn't used these days, not sure what it was ever needed for.

### CODE REVIEW

**Comment by diox** [comment]
lolwut r+

**Comment by wagnerand** [comment]
@diox sorry I just added another unused (and broken) file. Lots of cleanups...

File: functional_tests/test_validator.py [hunk 1]
```diff
@@ -18,8 +18,6 @@ def _validator(file_path, for_appversions=None, overrides=None):
     import validator
     import validator.constants
     js = os.environ.get('SPIDERMONKEY_INSTALLATION', 'js')
-    scripting.SPIDERMONKEY_INSTALLATION = js
-    validator.constants.SPIDERMONKEY_INSTALLATION = js
```
Comment (diox) [inline functional_tests/test_validator.py]: But then you don't need the line above either, right ?

Comment (diox) [inline functional_tests/test_validator.py]: err actually scratch that

Comment (diox) [inline functional_tests/test_validator.py]: so `validate()` does the right thing when you pass a custom `spidermonkey`, and that's why you're removing those references to `SPIDERMONKEY_INSTALLATION` ?

Comment (wagnerand) [inline functional_tests/test_validator.py]: `SPIDERMONKEY_INSTALLATION` isn't used in validator code at all. So it made no sense to me to set it.

**Comment by wagnerand** [comment]
Thank you!

**Pull request merged**